### ISSUE TITLE
notekit_0.2_amd64 has (false) unmet dependency on Ubuntu Jammy onwards.

### ISSUE THREAD

**Issue by timboxyz** [blackhole89/notekit#145]
There is a dependency on libjsoncpp1 >= 1.7.4. However Jammy uses libjsoncpp25 (1.9.5-3). They are the same base packages but from Jammy onwards the lib was renamed to have the "25" suffix instead of "1" and so is not recognised. The cross-build fails to take this into account..

**Comment by sp1ritCS** [comment]
Use the packages provided by OBS for Ubuntu 20.04 and onwards: https://software.opensuse.org//download.html?project=home%3Asp1rit%3Anotekit&package=notekit

**Comment by sp1ritCS** [comment]
It's in the same repo, google probably just doesn't pick it up in its index.

Alternatively, you can download the deb manually [here](https://download.opensuse.org/repositories/home:/sp1rit:/notekit/xUbuntu_22.04/amd64/) (Ubuntu 22.04) if you do not want to add the repo to receive updates.

**Comment by timboxyz** [comment]
TVM, If selecting the "grab packages directly" option, you don't get to see the repo, just links to the individual main deb so all the other files are hidden :(

**Comment by sp1ritCS** [comment]
yes, that is the intended behaviour of software-o-o. It's just supposed to offer the download options for the [notekit](https://build.opensuse.org/package/show/home:sp1rit:notekit/notekit) package built on OBS. The libclatexmath0 package has another link: https://software.opensuse.org//download.html?project=home%3Asp1rit%3Anotekit&package=libclatexmath0
